elements/tagging-question/src/tagging-question.js

```diff
@@ -65,8 +66,8 @@ class TaggingQuestion extends QuestionElement {
           border-radius: var(--ddd-radius-sm);
           box-sizing: border-box;
           background-color: light-dark(
-            var(--simple-colors-default-theme-grey-2),
-            var(--simple-colors-default-theme-grey-10)
+            var(--ddd-theme-default-coalyGray),
```

Comment:
no need for light-dark if this is going to be the same background color regaudless